### DISCUSSION TITLE
FEATURE: Add more html tag syntax

### DISCUSSION THREAD

**Discussion by Sleepy4k** [spatie/laravel-html#179] [Ideas]
maybe you can add a feature to use the br and li tag syntax to make it easier to work on (just in case)

**Comment by stijnvanouplines** [comment]
You can create an unordered list like this:
```php
html()->element('ul')->children([
    html()->element('li')->text('List item 1'),
    html()->element('li')->text('List item 2'),
    html()->element('li')->text('List item 3'),
]);
```

**Reply by Sleepy4k** [reply]
will it work on every tag? like fieldset tag and other

**Reply by stijnvanouplines** [reply]
Even a table with wrapping div:
```php
html()->element('div')->class('table')->children([
    html()->element('table')->children([
        html()->element('tr')->children([
            html()->element('th')->text('Header 1'),
            html()->element('th')->text('Header 2'),
            html()->element('th')->text('Header 3'),
        ]),
        html()->element('tr')->children([
            html()->element('td')->attribute('style', 'text-align:center')->text('Data 1'),
            html()->element('td')->attribute('style', 'text-align:center')->text('Data 2'),
            html()->element('td')->attribute('style', 'text-align:center')->text('Data 3'),
        ]),
    ]),
]);
```

**Reply by Sleepy4k** [reply]
> Even a table with wrapping div:
> 
> ```
> html()->element('div')->class('table')->children([
>     html()->element('table')->children([
>         html()->element('tr')->children([
>             html()->element('th')->text('Header 1'),
>             html()->element('th')->text('Header 2'),
>             html()->element('th')->text('Header 3'),
>         ]),
>         html()->element('tr')->children([
>             html()->element('td')->attribute('style', 'text-align:center')->text('Data 1'),
>             html()->element('td')->attribute('style', 'text-align:center')->text('Data 2'),
>             html()->element('td')->attribute('style', 'text-align:center')->text('Data 3'),
>         ]),
>     ]),
> ]);
> ```

ah i see, sorry for my miss understand element function, Thanks for helping me

**Reply by stijnvanouplines** [reply]
> Even a table with wrapping div:
> 
> ```
> html()->element('div')->class('table')->children([
>     html()->element('table')->children([
>         html()->element('tr')->children([
>             html()->element('th')->text('Header 1'),
>             html()->element('th')->text('Header 2'),
>             html()->element('th')->text('Header 3'),
>         ]),
>         html()->element('tr')->children([
>             html()->element('td')->attribute('style', 'text-align:center')->text('Data 1'),
>             html()->element('td')->attribute('style', 'text-align:center')->text('Data 2'),
>             html()->element('td')->attribute('style', 'text-align:center')->text('Data 3'),
>         ]),
>     ]),
> ]);
> ```

This will generate a table like this:
| Header 1 | Header 2 | Header 3 |
|:---------:|:---------:|:---------:|
| Data 1      | Data 2     | Data 3     |
